PhysicsEngine/PPolygonBuilder.java

```diff
@@ -0,0 +1,38 @@
+package PhysicsEngine;
+
+import java.util.ArrayList;
+import java.awt.Graphics;
+import java.awt.Color;
+import ToolBox.Vector;
+import ToolBox.Trigonometry;
+
+public class PPolygonBuilder
+{
+  // Fields representing the geometric and graphic structure of a polygon
+  private ArrayList<Vector> vertices = new ArrayList<Vector>();
+  private String name;
+  
+  public PPolygonBuilder() {
```

Comment:
Thanks for your pull request! However, please add Java docs so that implementations can know what the methods are for.